src/instructlab/cli/process/list.py

```diff
@@ -5,15 +5,30 @@
 
 # First Party
 from instructlab import clickext
+from instructlab.defaults import ILAB_PROCESS_STATUS
 from instructlab.process.process import list_processes
 from instructlab.utils import print_table
 
 
 @click.command(name="list")
+@click.option(
+    "--state",
+    type=click.Choice(
+        [
+            ILAB_PROCESS_STATUS.DONE,
+            ILAB_PROCESS_STATUS.RUNNING,
+            ILAB_PROCESS_STATUS.ERRORED,
+            "All",
+        ],
+        case_sensitive=False,
+    ),
+    default="All",
+    help="Filter processes with the specified state (default: All).",
+)
 @clickext.display_params
-def list():
+def list_and_print_processes(state: str):
```

Comment:
I understand that it's a good idea to not overload the keyword used for Python builtin, but what is the difference between "list" and "print"? Should it be just `list_processes`?

Reply:
I just refer to this one:
https://github.com/instructlab/instructlab/blob/d155da28311ce288db42f439cf7c8d444ba0b593/src/instructlab/model/list.py#L11